### PR TITLE
Provide learning rate multipliers for each layer

### PR DESCRIPTION
**Pull request by jrhupc**
This pull request allows the possibility to add a multiplication factor for learning rates in each layer. See Issue fchollet/keras/#414 for an explanation and discussion. Code is basically isolated from @AvantiShri at repo https://github.com/kundajelab/keras with the addition of a unit test.

### CODE REVIEW

**Comment by jrhupc** [comment]
First time user of automatic travis building tool. I added a commit to the pull request to solve the errors in the PEP8 check and in the second run the python3.4/Tensorflow check have failed (it was working before). Looking at the error it seems to be unrelated to my pull? :?

**Comment by aaronpolhamus** [comment]
In case anyone is interested, there was a bug relateing to the zero padding error that was rently worked out for the tensorflow backend implementation of keras. We've much master source with this repo here, allowing you to have all the fun you wanto to have in tensor flow with layer-specifc learning rates: https://github.com/badgley/keras

**Comment by DingKe** [comment]
Any update on that? What do you think about this feature @fchollet ? Learning rate scaling may be helpful in some cases, like this work about [binary network] (http://arxiv.org/pdf/1602.02830v3.pdf).

**Comment by perone** [comment]
This is also very useful for fine-tuning, any updates on that ?

**Comment by jeffzhengye** [comment]
up up up

**Comment by jmhessel** [comment]
Thanks for looking at this -- echoing my support for what would be a very useful feature.

**Comment by etdej** [comment]
A useful feature to come, especially for fine-tuning networks ! Thanks for looking at this

**Comment by jrhupc** [comment]
Updated pull request in #3004 to current version. I close this one.

**Comment by farzaa** [comment]
Any update on this?
